docs/DEVGUIDE.md

```diff
@@ -18,6 +28,26 @@ layout:
     ├── pkg            # Contains all non-"main" Go packages
     └── vendor         # Glide-managed dependencies (untracked)
 
+## Working on Issues
+
+Github does not allow non-maintainers to assign, or be assigned to, issues.
+As such non-maintainers can indicate their desire to work on (own) a particular
+issue by adding a comment to it of the form:
+
+	#dibs
+
+However, it is a good idea to discuss the issue, and your intent to work on it,
+with the other members via the slack channel to make sure there isn't some
+other work alread going on with respect to that issue.
```

Comment:
I've tried but I don't think non-maintainers are allowed to own issues. But perhaps I'm missing some aspect/feature of github